test/utils/__snapshots__/package-json.spec.ts.snap

```diff
@@ -7,7 +7,7 @@ Object {
     "@nestjs/common": "^6.7.2",
     "@nestjs/core": "^6.7.2",
     "@nestjs/platform-express": "^6.7.2",
-    "@sap/cloud-sdk-core": "^undefined",
+    "@sap-cloud-sdk/core": "^undefined",
```

Comment:
This is an indication that the dependency cant be found! If the comment above is fixed, this should be fixed when the tests are run...

Reply:
Please correct me. The comment above only defines the registry for `@sap`, so the scope `@sap-cloud-sdk` is not affected. Although I'm not sure why the dependency cannot be found, but it should be related to the previous comment?

Reply:
Actually I am surprised that the version is undefined even before the change... something is going wrong here.

I think your comment is correct but it might be that it matches all scopes that contain "@sap" instead of being an exact match. But I didnt find a source for that assumption yet.